src/transformers/models/roformer/tokenization_roformer.py

```diff
@@ -167,21 +182,21 @@ def __getstate__(self):
     def __setstate__(self, d):
         self.__dict__ = d
         try:
-            import rjieba
+            import jieba
         except ImportError:
             raise ImportError(
-                "You need to install rjieba to use RoFormerTokenizer."
-                "See https://pypi.org/project/rjieba/ for installation."
+                "You need to install jieba to use RoFormerTokenizer."
+                "See https://pypi.org/project/jieba/ for installation."
             )
-        self.jieba = rjieba
+        self.jieba = jieba
```

Comment:
If you already have the try: except block at the init of the tokenizer, is this required here?